Source/Core/VideoBackends/Vulkan/FramebufferManager.cpp

```diff
@@ -1109,7 +1135,7 @@ void FramebufferManager::CreatePokeVertices(std::vector<EFBPokeVertex>* destinat
                                             u32 y, float z, u32 color)
 {
   // Some devices don't support point sizes >1 (e.g. Adreno).
-  if (m_poke_primitive_topology == VK_PRIMITIVE_TOPOLOGY_TRIANGLE_STRIP)
+  if (m_poke_primitive != PrimitiveType::Points)
```

Comment:
Was this logic change intentional?

Reply:
Achieves the same thing. But again, for consistency I could reverse it.

Reply:
This would include triangles and lines in addition to triangle strips, right?  That's why I was concerned.

Reply:
Ah nm, I think I see your point now